src/screens/ComponentsScreen.tsx

```diff
@@ -1,12 +1,14 @@
 import * as Linking from 'expo-linking'
-import { ScrollView } from 'react-native'
+import { VStack, Heading, Divider, Button, ScrollView } from 'native-base'
 
-import { Button, Text } from '~components'
-import { useCallback, useNotifications, useTheme } from '~hooks'
+import { useCallback, useNotifications, useTranslation } from '~hooks'
+
+const headingSizes = ['xs', 'sm', 'md', 'lg', '2xl', '3xl', '4xl']
+const buttonVariants = ['solid', 'outline', 'subtle', 'ghost']
```

Comment:
You can add const assertion here and then you can remove 
```
{/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
{/* @ts-expect-error */}
```

```suggestion
const headingSizes = ['xs', 'sm', 'md', 'lg', '2xl', '3xl', '4xl'] as const
const buttonVariants = ['solid', 'outline', 'subtle', 'ghost'] as const
```